diff --git a/config.def.h b/config.def.h
index 877afab..6a1699f 100644
--- a/config.def.h
+++ b/config.def.h
@@ -84,44 +84,49 @@ static unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-};
 
+  /* 8 normal colors */
+  [0] = "#7e837c", /* black   */
+  [1] = "#fc796b", /* red     */
+  [2] = "#a8cb79", /* green   */
+  [3] = "#ed9c6a", /* yellow  */
+  [4] = "#bcbfb4", /* blue    */
+  [5] = "#d8bbbd", /* magenta */
+  [6] = "#a8cb79", /* cyan    */
+  [7] = "#fdd89a", /* white   */
+
+  /* 8 bright colors */
+  [8]  = "#7e837c", /* black   */
+  [9]  = "#fc796b", /* red     */
+  [10] = "#a8cb79", /* green   */
+  [11] = "#edc074", /* yellow  */
+  [12] = "#bcbfb4", /* blue    */
+  [13] = "#d8bbbd", /* magenta */
+  [14] = "#a8cb79", /* cyan    */
+  [15] = "#fdd89a", /* white   */
+
+  /* special colors */
+  [256] = "#3c4542", /* background */
+  [257] = "#fdd89a", /* foreground */
+};
 
 /*
  * Default colors (colorname index)
- * foreground, background, cursor, reverse cursor
+ * foreground, background, cursor
  */
-unsigned int defaultfg = 7;
-unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
+unsigned int defaultfg = 257;
+unsigned int defaultbg = 256;
+static unsigned int defaultcs = 257;
 static unsigned int defaultrcs = 257;
 
 /*
+ * Colors used, when the specific fg == defaultfg. So in reverse mode this
+ * will reverse too. Another logic would only make the simple feature too
+ * complex.
+ */
+unsigned int defaultitalic = 7;
+unsigned int defaultunderline = 7;
+/*
  * Default shape of cursor
  * 2: Block ("█")
  * 4: Underline ("_")
